src/rdbms_vnet/setup.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup, find_packages
+
+VERSION = "10.0.0"
```

Comment:
Why are you starting with such a high version?

Reply:
It's a remnant from private extension. Is it a blocker?

Reply:
Most extensions would start with something like `1.0.0` or `0.1.0` for preview.
It's strange starting off with `10.0.0` when there weren't any lower versions.
Would recommend changing but I'll leave the decision to you guys.